syntax = "proto3";

message SetTime
{
    int32 timestamp = 1;
}

message MusicControl
{
    int32 music_event = 1;
}

message SetWeather
{
    int32 timestamp = 1;
    int32 condition = 2;
    int32 temperature = 3;
    int32 temperature_min = 4;
    int32 temperature_max = 5;
    int32 humidity = 6;
}

message MessageNotification
{
    uint32 timestamp = 1;
    int32 type = 2;
    string from = 3;
    string title = 4;
    string text = 5;
}

message MusicInfo
{
    string artist = 1;
    string album = 2;
    string track = 3;
}